test/e2e/storage/csi_volumes.go

```diff
@@ -384,13 +383,10 @@ func initCSIgcePD(f *framework.Framework, config framework.VolumeTestConfig) csi
 }
 
 func (g *gcePDCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
-	nodeZone, ok := node.GetLabels()[kubeletapis.LabelZoneFailureDomain]
```

Comment:
we don't have topology support in csi yet. will this test potentially fail on multi-zone clusters now?

Reply:
yes, are all tests run in multi-zone? If I specify it then the integration test fails when I run with HEAD of the driver since we removed the zone parameter :S

Reply:
yes https://k8s-testgrid.appspot.com/sig-storage#gke-regional-serial

I think for now we need to skip this test in multi-zone environments, and add an [Feature] test for topology that we run in alpha clusters.

Reply:
add line for skipping in multi-zone